ibis/config.py

```diff
@@ -60,8 +60,24 @@ class Repr(BaseModel):
         description="The maximum number of characters to show in the `query` field repr of SQLQueryResult operations.",  # noqa: E501
     )
     show_types: bool = Field(
-        default=False,
-        description="Show the inferred type of value expressions in the repr.",
+        default=True,
+        description="Show the inferred type of value expressions.",
+    )
+
+    max_rows: int = Field(
+        default=10, description="Maximum rows to pretty print."
+    )
+    max_length: int = Field(
+        default=5,
+        description="Maximum length for pretty-printed arrays and maps.",
+    )
+    max_string: int | None = Field(
```

Comment:
Mild preference for `max_string_length` over `max_string`

Reply:
I had that originally too because I prefer it as well, but I ultimately kept the same name as the rich option to avoid confusion.